client/src/modules/list/components/Items/ListArchivedItem/ListArchivedItem.jsx

```diff
@@ -32,13 +33,15 @@ class ListArchivedItem extends PureComponent {
     const {
       currentUser: { name: userName },
       data: { _id: itemId, done, name },
+      intl: { formatMessage },
       match: {
         params: { id: listId }
       },
       restoreItem
     } = this.props;
+    const formattedName = formatName(userName, formatMessage);
```

Comment:
do we have to call `formatName` on the current User name? current user can not be anonymous...